src/desktop/apps/artist/client/views/market_data_summary.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { Contents } from 'reaction/Components/Artist/MarketDataSummary'
+import { ContextProvider } from 'reaction/Components/Artsy'
+
+function renderArtistMarketDataSummary(artistID) {
+  const elementID = 'market-data-summary-container-' + artistID
+  const mountPoint = document.getElementById(elementID)
+
+  if (mountPoint) {
```

Comment:
Oh yea, i guess this doesn't do anything if the element doesn't exist. 

Also, you don't need to wait until client-side javascript has fully loaded and started executing (and invoke from coffeescript). The way the artwork page insights container renders is invoked right from the assets file: https://github.com/artsy/force/blob/8624fb7891e199edd83e5b886b6d37776276bd1b/src/desktop/assets/artists_artworks.coffee#L12-L13  . It may be worth trying that if it feels quicker, esp since this is the header (which really should be rendered server-side...)

Reply:
Thinking more about this, I think we can render this server-side (due to @damassi most recent work), which might be preferable since this is the header of the page.

Given its behind a lab feature we can definitely still merge, but we should see how it feels and try it server-side if needed (would be a good practice for me to do that as well!).